drivers/google/compute_util.go

```diff
@@ -198,19 +219,87 @@ func (c *ComputeUtil) portsUsed() ([]string, error) {
 	return ports, nil
 }
 
-// openFirewallPorts configures the firewall to open docker and swarm ports.
-func (c *ComputeUtil) openFirewallPorts(d *Driver) error {
-	log.Infof("Opening firewall ports")
+// openInternalFirewallPorts configures a firewall rule for internal VPC network access only.
+func (c *ComputeUtil) openInternalFirewallPorts(d *Driver) error {
+
+	expectedPorts := []string{
+		"8443",
+		"179",
+		"5473",
+		"10256",
+		"10250",
+		"10251",
+		"10252",
+		"6443",
+		"2379",
+		"2380",
+		"4789",
+		"2376",
+		"9345",
```

Comment:
Do we still want to open this for k3s?

Reply:
We don't need it for k3s but we don't typically indicate the specific distribution when invoking rancher machine. For instance, IIUC creating a k3s cluster in AWS using the default security group definition will also open 9345. Since this is an internal-only firewall rule I think this is fine, but I'm open to other idea's if we think the port list needs to be more specific to the given distribution.

Reply:
From what I can tell the only ports not included in the current list which are documented are port `5001/tcp` (Spegel), `51820/udp` and `51821/udp` (Flannel Wireguard with IPv4 and v6). Since both of those are experimental / optional I haven't updated them in the list yet, but if we feel strongly about including them I'm not against it.